crates/bevy_math/src/bounding/bounded3d/mod.rs

```diff
@@ -451,6 +474,17 @@ impl BoundingVolume for BoundingSphere {
             },
         }
     }
+
+    #[inline(always)]
+    fn scale_around_center(&self, scale: Self::HalfSize) -> Self {
+        debug_assert!(scale >= 0.);
+        Self {
+            center: self.center,
+            sphere: Sphere {
+                radius: self.radius() * scale,
+            },
+        }
```

Comment:
This would be cleaner and what the `BoundingCircle` version already does
```suggestion
        Self::new(self.center, self.radius() * scale)
```

Reply:
Ah, I think I missed this since `grow` and `shrink` don't use `Self::new`.